example/thread_pool/psi.c

```diff
@@ -483,7 +483,7 @@ psi_include(TSCont contp, void *edata ATS_UNUSED)
         /* TSIOBufferStart allocates more blocks if required */
         block     = TSIOBufferStart(data->psi_buffer);
         ptr_block = TSIOBufferBlockWriteStart(block, &avail);
-        towrite   = MIN(ntodo, avail);
+        towrite   = ntodo < avail ? ntodo : avail;
```

Comment:
Why are you doing your own min instead of using std::min()?

Reply:
C...

Reply:
This is a C plugin.